clients/ui/frontend/package.json

```diff
@@ -15,6 +15,7 @@
     "build:bundle-profile": "webpack --config ./config/webpack.prod.js --profile --json > ./bundle.stats.json",
     "build:bundle-analyze": "webpack-bundle-analyzer ./bundle.stats.json",
     "build:prod": "webpack --config ./config/webpack.prod.js",
+    "build:prod:integrated": "DEPLOYMENT_MODE=integrated MOCK_AUTH=false webpack --config ./config/webpack.prod.js",
```

Comment:
@Griffin-Sullivan we have an issue here, we should talk about this tomorrow, not sure how to pull this off.

Reply:
Issue is that we control the build path at **build** time, meaning that we cannot control it with an environment variable at production time